api/db/db_models.py

```
@@ -1984,6 +1984,133 @@ def alter_db_column_type(migrator, table_name, column_name, new_column_type):
+    for table_name, column_name in TENANT_MODEL_ID_COLUMNS:
+        try:
+            table_present = DB.table_exists(table_name)
+        except Exception as ex:
+            logging.warning("Failed to inspect table %s while migrating tenant_*_id types: %s", table_name, ex)
+            continue
+        if not table_present:
+            continue
+
+        try:
+            col_type = _get_column_data_type(table_name, column_name)
+        except Exception as ex:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)